taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskEventCleaner.java

```diff
@@ -0,0 +1,59 @@
+package pro.taskana.adapter.systemconnector.camunda.api.impl;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestTemplate;
+
+import pro.taskana.adapter.impl.TaskanaTaskStarter;
+import pro.taskana.adapter.systemconnector.api.ReferencedTask;
+
+@Component
+public class CamundaTaskEventCleaner {
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TaskanaTaskStarter.class);
+
+    public void taskanaTasksHaveBeenCreatedFor(List<ReferencedTask> referencedTasks, String camundaSystemTaskEventUrl) {
```

Comment:
The method name "taskanaTasksHaveBeenCreatedFor" does not precisely describe what its implementation actually does (cleaning the outbox table).

Reply:
This is correct. But it is a method of the "abstract" adapter. There may be other implementations of the system connector that don't use an outbox table, but still need to take any action once the taskana task has been created ...

Reply:
I see, but even in that case I don't really get the intent of this method from its current name. "taskanaTasksHaveBeenCreatedFor" ....for what? The referencedTasks I assume. The way it is named i would also not expect a void method.